apps/blaze-dashboard/src/pages/setup/components/blaze-disabled.jsx

```diff
@@ -35,6 +36,7 @@ export default function BlazeDisabled() {
 				</li>
 				<li>{ translate( 'Scroll down to the Blaze section and toggle it to active.' ) }</li>
 			</ol>
+			<ConnectButton />
```

Comment:
NB: The Connect button and related component and style will be removed in favor of https://github.com/Automattic/wp-calypso/pull/90160 but leaving it here temporarily for easy testing until that PR gets merged and the changes made available to this PR.